content/id/docs/concepts/cluster-administration/federation.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Federation
```

Comment:
Should we do Federation or Federasi?

If we decide to use Federation, we will need to address this for previously translated pages (I remember some we use `kluster federasi`)

Reply:
Since this is related to kubernetes's term i'm not sure whether to chane it or not

Reply:
let's stick to federation for now